### DISCUSSION TITLE
GUI for gmail-processor

### DISCUSSION THREAD

**Discussion by PaysanBarbare** [ahochsteger/gmail-processor#205] [Ideas]
Hello,
Is it possible to add friendly GUI to modify the configuration ?

**Comment by ahochsteger** [comment]
In case you're not aware - there's currently a schema-assisted JSON config editor that supports content assist (using ctrl+space) to suggest allowed config parameters and values in the [Config Playground](https://ahochsteger.github.io/gmail-processor/playground).

In addition I'm thinking about adding a form-based UI using [react-jsonschema-form](https://rjsf-team.github.io/react-jsonschema-form/) if it works in combination with the site generator [Docusaurus](https://docusaurus.io/). But I'm unsure if this really will work using the generated JSON schema.

**Comment by ahochsteger** [comment]
Since there's already a UI in the playground that assists in creating the config (with syntax highlighting, validation and content assist) I'll close this discussion.
